contrib/checkbox-provider-ce-oem/units/strict-confinement/test-plan-strict-confinement.pxu

```diff
@@ -0,0 +1,36 @@
+id: dbus-warm-boot
+unit: test plan
+_name: Warm reboot test via dbus
+_description:
+    Warm reboot test by using dbus command.
+    Rely on test-strict-confinement SNAP to test.
+bootstrap_include:
+    com.canonical.certification::reboot-run-generator
+include:
+    dbus-warm-boot-boot
+    dbus-warm-boot-test
+
+
+id: dbus-cold-boot
```

Comment:
`dbus-cold-boot` test plan is not related SD PR, why we need this test plan?

Reply:
That is because I modified the name of the test plan, so it is treated as a new file.

Reply:
Understand